.github/workflows/e2e.yml

```diff
@@ -126,6 +126,12 @@ jobs:
             class: org.apache.streampark.e2e.cases.TeamManagementTest
           - name: MemberManagementTest
             class: org.apache.streampark.e2e.cases.MemberManagementTest
+          - name: Flink116OnYarnClustersTest
+            class: org.apache.streampark.e2e.cases.Flink116OnYarnClustersTest
```

Comment:
Flink supports multiple clusters: yarn-session, standalone, and k8s-session. However, only yarn-session is currently implemented. Is the name of this test case ambiguous?

Reply:
Maybe using `Flink116OnYarnSessionClustersTest`? Do you have better idea?

Reply:
This PR provides test cases for deploy and shutdown yarn-session cluster, How about Flink116OnYarnClusterDeployTest ?

Reply:
Okay. I'll fix it.